src/arkode/arkode.c

```diff
@@ -886,8 +886,8 @@ int ARKodeEvolve(void* arkode_mem, sunrealtype tout, N_Vector yout,
       }
 
       SUNLogInfo(ARK_LOGGER, "begin-step-attempt",
-                 "step = %li, tn = %" RSYM ", h = %" RSYM, ark_mem->nst + 1,
-                 ark_mem->tn, ark_mem->h);
+                 "step = %li, tn = " SUN_FORMAT_G ", h = " SUN_FORMAT_G,
+                 ark_mem->nst + 1, ark_mem->tn, ark_mem->h);
```

Comment:
I like that it looks like the new approach will be less "finicky" about requiring a space between the `"` and the format specifier (here `SUN_FORMAT_G`).